### DISCUSSION TITLE
Can i monitize on github pages

### DISCUSSION THREAD

**Discussion by impressstudio** [community/community#22094] [Other Feature Feedback, Questions, & Ideas]
<p>dear <a class="hashtag" href="/c/github-account-questions/42">#<span>github-account-questions</span></a> can i able to monitize on github</p>
<p>is it legal or illegal</p>
<p>plz reply</p>

**Comment by tmetz** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>The content of Pages is at the user’s discretion, as long as it’s legal and doesn’t otherwise violate the <a href="https://docs.github.com/en/github/site-policy/github-terms-of-service">Terms of Service</a>.</p>
<p>Some users may choose to include ads, donation buttons, ticket sales, or other ways of monetizing their projects, and that’s OK with us. Of course we hope folks will Keep It Classy™ and use this power for awesome!</p>

**Comment by teju0051** [comment]
can i add donation button in my github pages?? will i get banned?
